# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/stf/faad2/no-external-mp4v2.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- faad2/configure.in	2005-10-18 16:10:25.000000000 +0200
+++ faad2/configure.in	2005-10-18 16:10:31.000000000 +0200
@@ -67,7 +67,7 @@
         [HAVE_FPOS_T_POS])
 
 
-external_mp4v2=yes
+external_mp4v2=no
 AC_CHECK_LIB(mp4v2, MP4Create, , external_mp4v2=no, -lstdc++)
 AC_CHECK_LIB(mp4v2, MP4MetadataDelete, , external_mp4v2=no, -lstdc++)
 AC_CHECK_HEADER(mp4.h, , external_mp4v2=no)
